replay/model_handler.py

```diff
@@ -3,8 +3,6 @@
 import json
```

Comment:
lets run tests to ensure saving/loading works fine in local mode. merge lates changes please